sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactListCriteriaBuilder.java

```diff
@@ -152,6 +171,9 @@
 		case ContactJurisdictionDto.DISTRICT_UUID:
 			expressions.add(joins.getDistrict().get(District.NAME));
 			break;
+		case ContactIndexDto.ID:
+		case ContactIndexDto.CREATION_DATE:
+		case ContactIndexDto.COMPLETENESS:
```

Comment:
This will throw an exception whenever the user tries to sort by one of these columns; all three properties need to be moved to the first batch of "case" expressions in this switch statement instead